### ISSUE TITLE
maxRequests only the first loop?

### ISSUE THREAD

**Issue by mitch85** [aishek/axios-rate-limit#63]
Axios makes 2 requests in parallel, but after 30 seconds, it only makes one request at a time instead of two.

` 
  const axiosWithRateLimit = rateLimit(axios_update.create(), {
      maxRequests: 2,
      perMilliseconds: 30000
    })
`

**Comment by aishek** [comment]
Thank you for mentioning it! Since the lib have option to use several limits, I've fixed the bug in new PR with test included
